helm/flink-kubernetes-operator/templates/rbac.yaml

```diff
@@ -21,23 +21,14 @@ RBAC rules used to create the operator (cluster)role based on the scope
 */}}
 {{- define "flink-operator.rbacRules" }}
 rules:
-  - apiGroups:
-      - flink-operator
-    resources:
-      - "*"
-    verbs:
-      - "*"
   - apiGroups:
       - ""
     resources:
       - pods
       - services
-      - endpoints
```

Comment:
Not sure about the endpoints, we create quite a bunch:
```
(minikube:default)  ➜  flink-kubernetes-operator git:(FLINK-28228) k get endpoints
NAME                               ENDPOINTS                         AGE
basic-checkpoint-ha-example-rest   172.17.0.9:8081                   8s
basic-ingress                      172.17.0.7:6124,172.17.0.7:6123   16s
basic-ingress-rest                 172.17.0.7:8081                   16s
flink-operator-webhook-service     172.17.0.6:9443                   3m36s
```
Does the service permissions include this?

Reply:
As far as I can see, `services` RBAC covers it. Operator or flink native integration doesn't seem to be creating or managing endpoints directly. But I will double check.

Reply:
Discussed this offline with @mbalassi we don't manage endpoint directly, thus this one can be removed too

Reply:
Great, thanks